src/styles/card.css

```diff
@@ -0,0 +1,108 @@
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+
+  --min-element-margin: 16px;
+  --size-text-s: 10px;
+  --size-text-m: 12px;
+  --size-text-l: 14px;
+  --size-text-xl: 16px;
+  --color-gray: #616070;
+  --color-dark-blue: #060b35;
+  --color-blue: #00acdc;
+}
+
+.card__img-postion {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 40px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin-bottom: 4px;
+
+  font-size: var(--size-text-m);
+  font-weight: 500;
+  line-height: 18px;
+  color: var(--color-dark-blue);
+}
+
+.card__id-text {
+  font-weight: 400;
+  font-size: var(--size-text-s);
+  line-height: 14px;
+  color: var(--color-gray);
+
+  margin-bottom: var(--min-element-margin);
```

Comment:
I see margin rules somewhere placed at the top of the selector rules and somewhere at the bottom. Try to make your own approach with the same arrangement of the CSS rules through all files.